### ISSUE TITLE
Can't run Dropbox Uploader

### ISSUE THREAD

**Issue by gonzonia** [grandhack/pibroadcast#2]
When the installer gets to the step of Intsall/Syncing Dropbox it fails because  is can't change ownership in the boot partition 

chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/CHANGELOG.md': Operation not permitted
chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/Dockerfile': Operation not permitted
chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/Dockerfile.pi': Operation not permitted
chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/dropbox_uploader.sh': Operation not permitted
chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/dropShell.sh': Operation not permitted
chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/LICENSE': Operation not permitted
chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/README.md': Operation not permitted
chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/testUnit.sh': Operation not permitted
chown: changing ownership of '/boot/pibroadcast-scripts/Dropbox-Uploader/': Operation not permitted

I tried running the command with sudo and get the same result.

**Comment by grandhack** [comment]
Certainly looks like a lot has changed in the last 4 years with raspios. I might need to strip things down and re rewrite the core of this. Thanks for plugging away. I have several pi's (3, 3b+, 4) that u can use to rebuild with.

**Comment by gonzonia** [comment]
Yeah, I've hit a point where I don't know that I can go further (at least with DropBox) I tried moving the scripts folder to the home dir and running it. The chown works fine but then -
`  ~/pibroadcast-scripts/Dropbox-Uploader/dropbox_uploader.sh download /Media_Slides/ServerPi/ /home/pi/dropbox_media_slides/
-bash: /home/pi/pibroadcast-scripts/Dropbox-Uploader/dropbox_uploader.sh: Permission denied
`

**Comment by grandhack** [comment]
I am making this an optional add on. So if you don't need it, it will skip that section.

**Comment by gonzonia** [comment]
Sounds like a good idea.

**Comment by grandhack** [comment]
I started a testing branch. I think the installer for the Server and Client have been fixed. If you could clone the repo and then download the fresh_install.sh from the testing branch, and report back the results. That would greatly help.

**Comment by gonzonia** [comment]
Will do. I'll let you know if I have any questions.

**Comment by gonzonia** [comment]
Okay. Seemed to work well until near the end. I followed the original instruction and put it all in /boot but it looks like the script wanted it in /boot/pibroadcast so when it tried moving the scripts it didn't work.

**Comment by grandhack** [comment]
Okay. Try cloneing it to a regular users directory and then running it from there. I may change the initial install location away from /boot

**Comment by gonzonia** [comment]
Okay. Cloned to user dir. Switched into pibroadcast. Had to chmod +x fresh_install.sh to run it. Made the mistake of not running from user directory itself, but in pibroadcast so script was looking for /home/pi/pibroadcast/pibroadcast/pibroadcast-scripts/server_menu.sh

Ran again from user dir. I think it worked. I had some errors, but I think they were related to having just run it.

**Comment by grandhack** [comment]
That sound right. If you run it a second time it will throw some errors. That sounds like progress.

**Comment by gonzonia** [comment]
I'm installing a client next. Once I get that up, what's the best way to test the Server/Client together?

**Comment by grandhack** [comment]
On the server you will open a terminal window and begin the broadcast. If you are streaming something into the server over rtmp you should see it show up on the client. If you don’t stream anything, you might see a flash on the client.

**Comment by gonzonia** [comment]
Client had some issues with the Hostname. 
Output attached.
[Client Saved Output.txt](https://github.com/grandhack/pibroadcast/files/9331508/Client.Saved.Output.txt)
.

**Comment by grandhack** [comment]
Got it. Will keep pressing forward on debugging the client portion. Thanks for the logs

**Comment by gonzonia** [comment]
Client is a Pi Zero W. When I reboot the pi hostname was back to raspberrypi. Manually set hostname to EntrancePi1 and re-ran the install.  Logged in as piadmin and got the following - 
```
grep: /home/pi/.pibroadcast_settings: No such file or directory
grep: /home/pi/.pibroadcast_settings: No such file or directory
eth0: error fetching interface information: Device not found
eth0: error fetching interface information: Device not found
-bash: nmap: command not found
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
grep: /home/pi/.pibroadcast_settings: No such file or directory
grep: /home/pi/.pibroadcast_settings: No such file or directory
grep: /home/pi/.pibroadcast_settings: No such file or directory
grep: /home/pi/.pibroadcast_settings: No such file or directory
grep: /home/pi/.pibroadcast_settings: No such file or directory
grep: /home/pi/.pibroadcast_settings: No such file or directory
-bash: /home/piadmin/.bashrc: line 85: syntax error near unexpected token `2'
-bash: /home/piadmin/.bashrc: line 85: `			sudo sed -i "s/$feh_slide_time/$feh_pause_menu/g" /home/pi/.pibroadcast_settings >> 2>&1 $LOGFILE '
```
